build.sbt

```diff
@@ -67,13 +67,14 @@ lazy val root = (project in file("."))
 
 lazy val deps = {
   val awsSDKVersion = "2.11.4"
-  val pekkoVersion = "0.0.0+26610-defddc6a-SNAPSHOT"
-  val pekkoHttpVersion = "0.0.0+4311-07201517-SNAPSHOT"
+  val pekkoVersion = "0.0.0+26656-898c6970-SNAPSHOT"
+  val pekkoHttpVersion = "0.0.0+4345-fa1cb9cb-SNAPSHOT"
 
   Seq(
     "org.apache.pekko"        %% "pekko-stream"            % pekkoVersion,
     "org.apache.pekko"        %% "pekko-http"              % pekkoHttpVersion,
     "software.amazon.awssdk"  %  "http-client-spi"         % awsSDKVersion,
+    "org.scala-lang.modules"  %% "scala-java8-compat"      % "1.0.2",
```

Comment:
So the whole point of the new upstream changes to Pekko is it no longer provides `scala-java8-compat` for all versions of Scala. In our case we actually use these functions so we can just add it as a direct dependency.